splitio/engine/strategies/__init__.py

```diff
@@ -0,0 +1,153 @@
+import threading
```

Comment:
i'd suggest changing `strategies` to be a module instead of a package (maybe impstrategies.py? ) and put the base & concrete strategies there

Reply:
another options would be to create an `impressions` package. and 2 submodules. maybe one `utils.py` containing the hasher, observer, etc. and another one `strategies.py` with the strategies class hierearchy